src/editor/components/QueryEditorSection.tsx

```diff
@@ -3,12 +3,13 @@ import { InlineFormLabel } from '@grafana/ui';
 
 export interface QueryEditorSectionProps {
   label: string;
+  tooltip?: string;
 }
 
 export const QueryEditorSection: React.FC<PropsWithChildren<QueryEditorSectionProps>> = (props) => {
   return (
     <div className="gf-form">
-      <InlineFormLabel className="query-keyword" width={12}>
+      <InlineFormLabel className="query-keyword" width={12} tooltip={props.tooltip ?? ''}>
```

Comment:
not sure if the component works that way but there is the risk that this renders a tooltip icon but with no content. Better to leave it as `undefined`:

```suggestion
      <InlineFormLabel className="query-keyword" width={12} tooltip={props.tooltip}>
```

Reply:
Makes sense!